src/sql/src/statements/set_variables.rs

```diff
@@ -21,3 +23,25 @@ pub struct SetVariables {
     pub variable: ObjectName,
     pub value: Vec<Expr>,
 }
+
+impl SetVariables {
+    pub fn variable(&self) -> &ObjectName {
+        &self.variable
+    }
+
+    pub fn format_value(&self) -> String {
+        self.value
+            .iter()
+            .map(|expr| format!("{}", expr))
+            .collect::<Vec<_>>()
+            .join(", ")
```

Comment:
I guess our set value should be always one element. But this is out of the scope of this PR. Just to comment it clearly.